modules/sandboxed-containers-uninstalling-kata-runtime.adoc

```diff
@@ -17,4 +17,4 @@ This section describes how to remove and uninstall the `kata` runtime and all it
 oc delete kataconfig <KataConfig_CR_Name>
 ----
 
-The {sandboxed-containers} removes all resources that were initially created to enable the runtime on your cluster. After you run the command above, your cluster is restored to the state prior to the installation process.
+The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster. After you run the command above, your cluster is restored to the state prior to the installation process.
```

Comment:
@jensfr @bpradipt can you ack this behavior?

Reply:
@zanetworker This was from the initial commit. I haven't updated this section yet. So, if this content changed in the Google Doc it is not _yet_ reflected.

Reply:
@zanetworker @sagidlow ack from my side